pkg/www/retext/src/components/TextRenderer.vue

```diff
@@ -142,21 +142,30 @@
                     currentWord = this.words(indices[0], indices[1])[indices[2]];
                 }
 
-
                 let div = createDiv(e.clientX, e.clientY);
                 div.innerText = "\"" +  selectedWords.reduce((acc, val) => {return acc + " " + val;}) + "\"";
 
                 document.body.appendChild(div);
 
                 let move = (e) => {
-                    div.style.left = e.clientX + "px";
-                    div.style.top = e.clientY + "px";
+                    div.style.left = (1 + e.clientX) + "px";
+                    div.style.top = (e.clientY) + "px";
                 }
 
                 let remove = () => {
                     div.remove()
                     document.removeEventListener("mouseup", remove);
                     document.removeEventListener("mousemove", move);
+
+                    let words = JSON.parse(JSON.stringify(this.dragTool));
```

Comment:
I suppose this makes a copy of the dragTool object? There must be a method that will make that more explicity

Reply:
keep an eye on: https://stackoverflow.com/questions/122102/what-is-the-most-efficient-way-to-deep-clone-an-object-in-javascript

apparently, they're working on a `structuredClone()` in Node that might get to browsers.

The other options are slower, shallow copies, or require a library.